File: Array/Sorting/Bubble-Sorting/BubbleSort.py
# Input the size of the array
n = int(input("Enter the size of the array: "))

# Input the array elements
arr = []
print(f"Enter {n} elements:")
for _ in range(n):
    arr.append(int(input()))

# Print original array
print("Original array:", arr)

# Bubble sort (ascending order)
for i in range(n - 1):
    for j in range(n - 1 - i):
        if arr[j] > arr[j + 1]:
            arr[j], arr[j + 1] = arr[j + 1], arr[j]

# Print sorted array
print("Sorted array:", arr)

# Bubble Sort algorithm implemented in Python

File: Array/Sorting/Bubble-Sorting/BubbleSort.java
import java.util.Scanner;

public class BubbleSort {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            // Get the size of the array
            System.out.print("Enter the size of the array: ");
            int n = sc.nextInt();

            int[] arr = new int[n];

            // Input the elements
            System.out.println("Enter " + n + " elements:");
            for (int i = 0; i < n; i++) {
                arr[i] = sc.nextInt();
            }

            // Print the original array
            System.out.print("Original array: ");
            for (int i = 0; i < n; i++) {
                System.out.print(arr[i] + " ");
            }
            System.out.println();

            // Bubble Sort (ascending order)
            for (int i = 0; i < n - 1; i++) {
                for (int j = 0; j < n - 1 - i; j++) {
                    if (arr[j] > arr[j + 1]) {
                        int temp = arr[j];
                        arr[j] = arr[j + 1];
                        arr[j + 1] = temp;
                    }
                }
            }

            // Print the sorted array
            System.out.print("Sorted array: ");
            for (int i = 0; i < n; i++) {
                System.out.print(arr[i] + " ");
            }
            System.out.println();
        }
    }
}
// Bubble Sort algorithm implemented in Java